src/components/table/AltinnSummaryTable.tsx

```diff
@@ -21,33 +15,15 @@ export interface IAltinnSummaryTableProps {
   summaryDataObject: SummaryDataObject;
 }
 
-const InnerRow = ({ name, prop }: InnerRowProps) => (
-  <TableRow className={classes.tableRow}>
-    <TableCell
-      padding='none'
-      className={classes.tableCell}
-    >
-      <Typography variant='body1'>{name}:</Typography>
-    </TableCell>
-    <TableCell
-      padding='none'
-      className={cn(classes.tableCell, { ['no-visual-testing']: prop.hideFromVisualTesting })}
-    >
-      <Typography variant='body1'>{prop.value}</Typography>
-    </TableCell>
-  </TableRow>
-);
-
-export const AltinnSummaryTable = (props: IAltinnSummaryTableProps) => (
-  <Table className={classes.instanceMetaData}>
-    <TableBody>
-      {Object.keys(props.summaryDataObject).map((name) => (
-        <InnerRow
-          key={name}
-          name={name}
-          prop={props.summaryDataObject[name]}
-        />
+export const AltinnSummaryTable = ({ summaryDataObject }: IAltinnSummaryTableProps) => (
+  <table className={classes.table}>
```

Comment:
Hvorfor ikke Designsystemet `Table`-komponent? 😄

Reply:
Fordi denne tabellen, rent utseendemessig, er heeelt basic. Det er såvidt det er en tabell engang. Jeg prøvde først å bare lage denne visningen med noen div-elementer og css grid, men det ble mye knot. Alt ble veldig mye enklere når jeg bare gjorde den om til en helt vanlig table. Med designsystemet måtte jeg ha overstyrt massevis av styling for å få den til å se ut som en simpel tabell (altså gitt at den skulle se lik ut), så det ville jo vært mot hensikten.

Jeg synes godt vi kan bytte ut denne greia med noe annet i fremtiden, men jeg er usikker på om tabellen fra designsystemet er riktig det heller. Men ingen poeng i å endre utseendet på den nå i forbindelse med fjerning av MUI.